lxd/main_init.go

```diff
@@ -555,8 +542,29 @@ func (cmd *CmdInit) initConfig(client lxd.ContainerServer, config map[string]int
 		return err
 	}
 
-	server.Config = config
+	// If the auto-update interval is already set to a non-zero
+	// value, and we're being requested to change it to a value
+	// other than zero, we don't want to overwrite it, so we'just
+	// delete it from the desired config.
+	if curVal, ok := server.Config["images.auto_update_interval"]; ok && curVal != "0" {
```

Comment:
I think I might have mentioned this before and you don't need to change this now but for the future:

```
curVal, ok := server.Config["images.auto_update_interval"]
if ok && curVal != "0" {
        // something
}
```

to keep the clode nice an uncluttered. :)